### ISSUE TITLE
Ultima Icon display issue with p-confirmDialog

### ISSUE THREAD

**Issue by smitpsanghavi** [primefaces/ultima-ng#62]
There seems to be some problem for displaying icons in confirmDialog component. According to the documentation for Ultima, I can use the ui-icon-* notation to specify an icon. I tried to display an icon in confirmDialog and this is what I got:

```html
<p-confirmDialog icon="ui-icon-delete" header="Delete Confirmation"></p-confirmDialog>
```
![dialog-no-icon](https://cloud.githubusercontent.com/assets/10339097/26308665/bdd8ab74-3f17-11e7-8caf-11392559a69e.JPG)

HTML in DOM:
```html
<div class="ui-dialog-content ui-widget-content">
	<i class="ui-icon-delete"></i>
	<span class="ui-confirmdialog-message">Deleting 'Projects Head'. Confirm?</span>
</div>
```

I now tried using FontAwesome icons and it got displayed. Why do I need Font Awesome to display the delete icon or for the matter of fact any other icon in confirmDialog?
```html
<p-confirmDialog icon="fa ui-icon-delete" header="Delete Confirmation"></p-confirmDialog>
```
![dialog-fa-icon](https://cloud.githubusercontent.com/assets/10339097/26308704/e833dfe2-3f17-11e7-804b-5b76885710fd.JPG)
HTML in DOM:
```html
<div class="ui-dialog-content ui-widget-content">
	<i class="fa ui-icon-delete"></i>
	<span class="ui-confirmdialog-message">Deleting 'Projects Head'. Confirm?</span>
</div>
```

Can @cagataycivici explain why I am having such a behavior?

**Comment by alygueye** [comment]
I get the same problem when using ui-icon-*.
Is there any solution to this.
thanks

**Comment by Mrtcndkn** [comment]
It is related to PrimeNG we fixed this for PrimeNG 4.1.1 release which will be released today.

https://github.com/primefaces/primeng/issues/3444
